stdlib/string.gr

```diff
@@ -329,13 +328,13 @@ export let slice = (start: Number, to: Number, s: String) => {
 
   let mut start = WasmI32.fromGrain(start)
   if ((start & 1n) != 1n) {
-    throwError(Errors._GRAIN_ERR_INVALID_ARGUMENT, start, 0n)
+    throw InvalidArgument("invalid start index")
```

Comment:
Do we want to capitalize the error messages in this file to match most of the other messages that have been adjusted?
```suggestion
    throw InvalidArgument("Invalid start index")
```

Reply:
If we decide we're going that route!